src/character.cpp

```diff
@@ -4417,17 +4417,17 @@ void Character::update_needs( int rate_multiplier )
     if( in_vehicle && get_size() == creature_size::huge &&
         !( has_trait( trait_NOPAIN ) || has_effect( effect_narcosis ) ) ) {
         vehicle *veh = veh_pointer_or_null( get_map().veh_at( pos() ) );
-        // it's painful to work the controls, but passengers in open topped vehicles are fine
-        if( veh && ( veh->enclosed_at( pos() ) || veh->player_in_control( *this ) ) ) {
+        // whether or not you are working the controls you will recieve pain if you are in a limited_space area.
```

Comment:
If we're going to have regular and huge variants, simply disallow use of the mismatching controls.

Reply:
I am attempting to do that, running into errors obviously. Let me know if you see a better suggestion to what I'm currently trying (and failing) to compile. Thank you!